### PR TITLE
Add constituency model

### PR DESCRIPTION
**Pull request by 0sc**
This PR adds the constituency model

### CODE REVIEW

**Comment by silentworks** [comment]
Good job @0sc. I cannot merge this because the build failed, there error is:

```
test/models/constituency_test.rb:1:9: C: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
require 'test_helper'
```

Please fix and update your PR and I will get it merged in.

**Comment by 0sc** [comment]
Ahh, good catch. I'll make the update.

File: db/migrate/20171009134732_create_constituencies.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateConstituencies < ActiveRecord::Migration[5.0]
+  def change
+    create_table :constituencies do |t|
```
Comment (yawboakye) [inline db/migrate/20171009134732_create_constituencies.rb]: Use `uuid` for constituency id

**Review by yawboakye — CHANGES_REQUESTED** [review]

File: db/migrate/20171009134732_create_constituencies.rb (hunk 1 above)
Comment (0sc) [reply db/migrate/20171009134732_create_constituencies.rb]: Done @yawboakye

**Review by yawboakye — APPROVED** [review]
LGTM cc/ @bubunyo can you have another pass? Thank you.

**Review by bubunyo — APPROVED** [review]
Seems legit.

**Pull request merged**